device/bgp/bgp.yaml

```diff
@@ -114,3 +114,147 @@ components:
         name:
           x-include: ../../common/common.yaml#/components/schemas/Named.Object/properties/name
           x-field-uid: 11
+
+    Device.Bgp.MessageHeaderError:
+      description: >-
+        All errors detected while processing the Message Header are indicated by sending the NOTIFICATION message 
+        with the Error Code-Message Header Error. The Error Subcode elaborates on the specific nature of the error.
+      type: object
+      properties:
+        subcode:
+          description: >-
+            The Error Subcode, the specific type of error encountered during Message Header processing.
```

Comment:
The Error Subcode indicates the specific ...

Reply:
updated